Modules/Core/Common/include/itkMetaDataObject.hxx

```diff
@@ -59,11 +59,45 @@ MetaDataObject<MetaDataObjectType>::SetMetaDataObjectValue(const MetaDataObjectT
   Self::Assign(m_MetaDataObjectValue, newValue);
 }
 
+namespace
+{
```

Comment:
Please don't open an unnamed namespace in a header file (*.h, *.hxx)

Reply:
Looks like a good excuse for an immediate follow-up? 😄 Is there also some chance for you tackle `std::vector` printing?

Reply:
@N-Dekker What is the alternative suggestion?  Should move the classes to ITK's meta programming sub namespace?

Reply:
> Should move the classes to ITK's meta programming sub namespace?

Good question... is the intention to have `has_Print` and `has_output_operator` as part of the public interface, for end-users? Then they could go into the meta programming sub-namespace. Or they could just remain in the root of the `itk` namespace. But then they should still be moved from the ".hxx" file into a ".h" file

Otherwise, is the intention to make clear to users that `has_Print` and `has_output_operator` are just implementation details? Then we could place them into some implementation-detail-namespace. We could for example name "itkMetaDataObjectImplementationDetail"

A third option: If you want to entirely hide them away from the end-users, you could probably make them privately nested types of MetaDataObject.